### ISSUE TITLE
Repo questions

### ISSUE THREAD

**Issue by DraakUSA** [otem/Raspberry-Pi-Looper-synth-drum-thing#3]
From looking at the repo, I am assuming that piLoopControl.ino is the code for the Teensy, and that everything in the piLooper directory goes on the RaspberryPi.  What code are you running on the RaspberryPi?

**Comment by otem** [comment]
You are correct. The piLooper directory contains all the pure data code. Just open up piLooper.pd

**Comment by DraakUSA** [comment]
It took me a while but I now believe i understand that "pure data" is the name of the application I need to install on the Pi which uses the ".pd" files, correct?

**Comment by DubbyLabby** [comment]
Yes @DraakUSA.

**Comment by dennischrist** [comment]
Hi, do you also have a wire diagram how to connect all the components?
